src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -42,16 +42,20 @@
 #include "utils/gen_hash.h"
 #include "utils/helpers.h"
 
-namespace sdl_rpc_plugin {
-using namespace application_manager;
+const char* kInvalidImageWarningInfo = "Requested image(s) not found.";
 
+namespace sdl_rpc_plugin {
 namespace commands {
 
+namespace am = application_manager;
```

Comment:
These abbreviations are normally only done in test files, I think that the full interface names should be used in this file.

Reply:
@jacobkeeler agree. These changes were reverted in 5f0bb3cd3d606adf9659e38e3348c8ed8afd3735